htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -72,6 +72,8 @@ INSERT INTO llx_accounting_system (fk_country, pcg_version, label, active) VALUE
 ALTER TABLE llx_product_lot ADD COLUMN eol_date datetime NULL;
 ALTER TABLE llx_product_lot ADD COLUMN manufacturing_date datetime NULL;
 ALTER TABLE llx_product_lot ADD COLUMN scrapping_date datetime NULL;
+ALTER TABLE llx_product_lot ADD COLUMN commissionning_date datetime NULL;
```

Comment:
This fields has been voluntary discarded (its use and position in this table songs wrong)